apps/ai-content-generator/src/components/app/dialog/common-generator/CommonGenerator.tsx

```diff
@@ -17,11 +18,12 @@ const initialParameters: GeneratorParameters = {
   sourceField: '',
   outputField: '',
   originalText: '',
+  canGenerateTextFromField: false,
 };
 
 const CommonGenerator = (props: FeatureComponentProps) => {
   const { isTranslate, isTitle } = props;
-  const { setProviderData } = useContext(GeneratorContext);
+  const { setProviderData, feature } = useContext(GeneratorContext);
```

Comment:
Since feature lives in context I think it'd be cleaner to just call useContext in the `NoFieldsSelectedMessage`